event.c

```diff
@@ -700,7 +700,7 @@ static void event_ffmpeg_timelapse(struct context *cnt,
     if (!cnt->ffmpeg_timelapse) {
```

Comment:
I'd possibly add a few notes to the commit message to this one about why swf is being named etc.